tests/system/src/test/java/test/robot/javafx/scene/NodeInitializationStressTest.java

```diff
@@ -0,0 +1,979 @@
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8348100") // FIX
+    @Test
+    public void tooltip() {
+        assumeFalse(SKIP_TEST);
+        // TODO will have a better test in JDK-8348100
+    }
+
+    @Test
+    public void treeTableView() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            TreeTableView<String> c = new TreeTableView<>();
+            c.setSkin(new TreeTableViewSkin(c));
+            c.setRoot(createRoot());
+            TreeTableColumn<String,String> col = new TreeTableColumn<>("TreeTable");
+            col.setCellValueFactory((cdf) -> {
+                Object v = cdf.getValue();
+                return new SimpleObjectProperty(v.toString());
+            });
+            c.getColumns().add(col);
+            return c;
+        }, (c) -> {
+            c.setRoot(createRoot());
+            accessControl(c);
+        });
+    }
+
+    @Test
+    public void treeView() {
+        assumeFalse(SKIP_TEST);
+        Supplier<TreeItem<String>> gen = () -> {
+            TreeItem<String> root = new TreeItem<>(null);
+            return root;
+        };
+
+        test(() -> {
+            TreeView<String> c = new TreeView<>();
+            c.setSkin(new TreeViewSkin(c));
+            c.setRoot(createRoot());
+            return c;
+        }, (c) -> {
+            c.setRoot(createRoot());
+            accessControl(c);
+        });
+    }
+
+    private void accessTextInputControl(TextInputControl c) {
+        accessControl(c);
+        c.setPromptText("yo");
+        c.setText(nextString());
+        c.prefHeight(-1);
+    }
+
+    private void accessChart(Chart c) {
+        String title = c.getClass().getSimpleName();
+        c.setTitle(title);
+        c.setAnimated(true);
+        accessRegion(c);
+    }
+
+    private void accessControl(Control c) {
+        accessRegion(c);
+        c.getCssMetaData();
+    }
+
+    private void accessNode(Node c) {
+        c.setFocusTraversable(true);
+        c.requestFocus();
+        c.toFront();
+    }
+
+    private void accessRegion(Region c) {
+        accessNode(c);
+        c.prefHeight(-1);
+        c.prefWidth(-1);
+        c.setPrefWidth(nextBoolean(0.1) ? 20 : 100);
+        c.setPrefHeight(nextBoolean(0.1) ? 20 : 100);
+    }
+
+    private <T extends Node> void test(Supplier<T> generator, Consumer<T> operation) {
+        AtomicReference<T> ref = new AtomicReference();
+        runAndWait(() -> {
+            T n = generator.get();
+            ref.set(n);
+        });
+        T visibleNode = ref.get();
+        String title = visibleNode.getId();
+        if (title == null) {
+            title = visibleNode.getClass().getSimpleName();
+        }
+
+        setTitle(title);
+        setContent(visibleNode);
+
+        int threadCount = 1 + Runtime.getRuntime().availableProcessors() * 2;
+        AtomicBoolean running = new AtomicBoolean(true);
+        int additionalThreads = 2; // jiggler + tight loop
```

Comment:
I wonder if it is worth having more than 1 "tight loop" thread? Maybe dedicate 1/2 of the threads to a tight loop (`generator` in the loop) and 1/2 to access (`generator` once and `operation` in the loop) This is something that you could consider for a future improvement,

Reply:
this is an interesting idea, esp. for intermittent tests.  I'll keep this as is for now, but will use the idea if we encounter more stubborn tests.